scripts/create_newcase

```diff
@@ -170,6 +170,8 @@ def _main_func():
 
         # Write out the case files
         case.flush(flushall=True)
+        if user_mods_dir is not None and user_mods_dir.startswith(".."):
```

Comment:
I tried that but there are three possiblities:   1. The path is already absolute, 2. the path is relative to the current directory.   The path is incomplete and the root directory is found in config_paths.xml    Calling abspath everytime breaks #3 but I cant think of a case of 2 where the above wouldn't work.